cloudapp/src/app/column-definitions.ts

```diff
@@ -1,4 +1,4 @@
-import { RequestedResource } from './print-slip-report'
+import { RequestedResource } from "./requested-resources-service/requested-resources-service"
```

Comment:
Nitpick. Please use single instead of double quotes.

Reply:
Resolved with commit b075a3c